src/main/java/uk/gov/hmcts/reform/pip/account/management/database/UserRepository.java

```diff
@@ -34,9 +34,12 @@ List<PiUser> findExistingByProvenanceId(@Param("provUserId") String provenanceUs
         + " * :daysAgo AND user_provenance = 'PI_AAD' AND roles = 'VERIFIED'", nativeQuery = true)
     List<PiUser> findVerifiedUsersByLastVerifiedDate(@Param("daysAgo") int daysSinceLastVerified);
 
-    @Query(value = "SELECT * FROM pi_user WHERE CAST(last_signed_in_date AS DATE) = CURRENT_DATE - (interval '1' day)"
-        + " * :daysAgo AND roles <> 'VERIFIED' AND user_provenance = 'PI_AAD'", nativeQuery = true)
-    List<PiUser> findAadAdminUsersByLastSignedInDate(@Param("daysAgo") int daysSinceLastSignedIn);
+    @Query(value = "SELECT * FROM pi_user WHERE (user_provenance = 'PI_AAD' AND roles <> 'VERIFIED' AND "
+        + "CAST(last_signed_in_date AS DATE) <= CURRENT_DATE - (interval '1' day) * :aadDays) OR "
```

Comment:
The <= here I think will cause an issue for notifications as I think they'll get an email every day during the 14 day notification period.

This will work for Deletion though, and I think adds weight to to the other point about splitting them out

Reply:
As above :)